src/automata.jl

```diff
@@ -149,6 +149,44 @@ function _skip_next(c::RealTimeScheduling.MissRow)
     return T, μ
 end
 
+function _skip_next(c::RealTimeScheduling.MeetAny)
+    if c.meet == c.window
+        T, μ = _skip_next(RealTimeScheduling.MissRow(0))
+    elseif c.meet == 0
+        T, μ = _skip_next()
+    else
+        L = 2^c.window
+		T = zeros(Union{Missing, Int64}, (L, 2))
```

Comment:
Please make sure the indentation uses four spaces per level, rather than hard tabs, as mentioned in the [Julia style guide](https://docs.julialang.org/en/v1/manual/style-guide/)